bokeh/_templates/autoload_js.js

```diff
@@ -76,7 +83,7 @@ calls it with the rendered model.
     console.log("Bokeh: BokehJS loaded, going straight to plotting");
```

Comment:
We can't always unconditionally test for widgets tho, the entire point is so that folks not using widgets don't have to pay for them. Full BokehJS is *huge*, and skipping widgets can trim almost an entire MB of the page load.